examples/cluster_bomb.rs

```diff
@@ -1,9 +1,11 @@
 extern crate rand;
```

Comment:
Opted for the quick approach to get things passing with the parallel feature disabled by just panicking.

I thought that the work of making the examples work in parallel and non-parallel could be left for the future since these wouldn't have worked in `not(parallel)` before anyway